src/components/ServiceCatalog/ServiceCard.tsx

```diff
@@ -55,7 +57,7 @@ const ServiceCard = ({
       </CardHeader>
       <CardBody>{description}</CardBody>
       <CardFooter>
-        {helperText != undefined ? helperText(status) : ''}
+        {helperText != undefined ? helperText(status || '', provisioningLabel || '') : ''}
```

Comment:
@mfrancisc this is difficult to read. Can we set the default value of `status` and `helperText` above to `''`? That way if it is not passed in we default to `''`. 

You'd have:
```ts
const ServiceCard = ({
  title,
  subtitle,
  description,
  iconUrl,
  learnMoreUrl,
  buttonOptions,
  buttonsFunc,
  status = '',
  helperText,
  provisioningLabel = '',
}: Props) => {
  return ( ...
```

It is important to note however any falsey value passed in will override that default.

Reply:
thanks for the suggestion, I've applied that in https://github.com/RedHatInsights/developer-sandbox-ui/pull/67/commits/836ef45b9442fd2c459a735a5b09f710e7b1c76e